src/main/java/org/web3j/console/project/kotlin/KotlinProject.java

```diff
@@ -107,9 +107,9 @@ public KotlinTemplateProvider getTemplateProvider() {
         }
 
         if (withSampleCode) {
-            templateBuilder.withMainJavaClass("Kotlin.template");
+            templateBuilder.withMainKotlinClass("Kotlin.template");
         } else {
-            templateBuilder.withMainJavaClass("EmptyTemplate.java");
+            templateBuilder.withMainKotlinClass("EmptyKotlin.template");
```

Comment:
For Java, can we follow a similar naming standard for the template?

Currently it seems to be called EmptyTemplate.java, so rename it to EmptyJava.template I think.

Reply:
done